### PR TITLE
scan-rn-doc-417

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.17

Jira: [OCPBUGS-33699](https://issues.redhat.com/browse/OCPBUGS-33699)

Link to docs preview:
[RN 4.17 BUG FIXES](https://82691--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-bug-fixes_release-notes)

### CODE REVIEW

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Review by JoeAldinger — APPROVED** [review]

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -564,10 +564,6 @@ An {product-title} cluster with multi-architecture compute machines supports com
 * xref:../installing/installing_aws/upi/installing-aws-multiarch-support-upi.adoc#installing-a-cluster-with-multiarch-support_upi-aws-multiarch-support[Installing a cluster with multi-architecture support ({aws-short}: User-provisioned infrastructure)]
 * xref:../installing/installing_gcp/installing-gcp-multiarch-support.html#installing-a-cluster-with-multiarch-support_installing-gcp-multiarch-support[Installing a cluster with multi-architecture support ({gcp-short})]
 
-==== Deploy {azure-short} in the Spain region
```
Comment (kalexand-rh) [inline release_notes/ocp-4-17-release-notes.adoc:567]: @dfitzmau, did you mean to remove this RN?

Comment (dfitzmau) [reply release_notes/ocp-4-17-release-notes.adoc:567]: Thanks for the comment. Yes, as it's a dupe. See the following note: 
Screenshot from 2024-09-30 13-54-25

**Pull request merged**